app/assets/javascripts/collections/user.js

```diff
@@ -0,0 +1,14 @@
+
+(function(App) {
+
+  'use strict';
+
+  App.Collection.User = App.Collection.Table.extend({
+
+    // url: 'admin/users.json'
+
+    url: '/users.json'
```

Comment:
Last time I checked it put it returned 1 data. That's not very useful. That's why we have our own examples. I'll remove it when we have real data in the API.